apl/tabular-operators/order-operator.mdx

```diff
@@ -1,33 +1,146 @@
+The `order` operator in Axiom Processing Language (APL) allows you to sort the rows of a result set by one or more specified columns. You can use this operator to organize data for easier interpretation, prioritize specific values, or prepare data for subsequent analysis steps. The `order` operator is particularly useful when working with logs, telemetry data, or any dataset where ranking or sorting by values (such as time, status, or user ID) is necessary.
 
-## Syntax
+## For users of other query languages
+
+If you come from other query languages, this section explains how to adjust your existing queries to achieve the same results in APL.
+
+<AccordionGroup>
+<Accordion title="Splunk SPL users">
+
+In Splunk SPL, the equivalent operator to `order` is `sort`. SPL uses a similar syntax to APL but with some differences. In SPL, `sort` allows both ascending (`asc`) and descending (`desc`) sorting, while in APL, you achieve sorting using the `asc()` and `desc()` functions for columns.
+
+<CodeGroup>
+```splunk Splunk example
+| sort - _time
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| order by _time desc
+```
+</CodeGroup>
+
+</Accordion>
+<Accordion title="ANSI SQL users">
+
+In ANSI SQL, the equivalent of `order` is `ORDER BY`. SQL uses `ASC` for ascending and `DESC` for descending order. In APL, sorting works similarly, with the `asc()` and `desc()` functions added around column names to specify the order.
+
+<CodeGroup>
+```sql SQL example
+SELECT * FROM logs ORDER BY _time DESC;
+```
+
+```kusto APL equivalent
+['sample-http-logs']
+| order by _time desc
+```
+</CodeGroup>
+
+</Accordion>
+</AccordionGroup>
+
+## Usage
+
+### Syntax
+
+```kusto
+| order by ColumnName [asc | desc], ColumnName [asc | desc]
+```
+
+### Parameters
+
+- `ColumnName`: The name of the column by which to sort.
+- `asc`: Sorts the column in ascending order.
+- `desc`: Sorts the column in descending order.
+
+### Returns
+
+The `order` operator returns the input dataset, sorted according to the specified columns and order (ascending or descending). If multiple columns are specified, sorting is done based on the first column, then by the second if values in the first column are equal, and so on.
+
+## Use case examples
+
+<Tabs>
+<Tab title="Log analysis">
+
+In this example, you sort HTTP logs by request duration in descending order to prioritize the longest requests.
+
+**Query**
 
 ```kusto
-| order by Field [asc], Field2 [desc]
+['sample-http-logs']
+| order by req_duration_ms desc
 ```
 
-[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%5Cn%7C%20order%20by%20content_type%20asc%2C%20is_tls%20desc%22%2C%22queryOptions%22%3A%7B%22quickRange%22%3A%2230d%22%7D%7D)
+[Run in Playground](https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%20%7C%20order%20by%20req_duration_ms%20desc%22%7D)
```

Comment:
![image](https://github.com/user-attachments/assets/839efb63-c46b-4801-80e0-9584e61faecd)

Reply:
No error for me
https://play.axiom.co/axiom-play-qf1k/explorer?initForm=%7B%22apl%22%3A%22%5B%27sample-http-logs%27%5D%20%7C%20order%20by%20req_duration_ms%20desc%22%7D